tests/src/cgeo/geocaching/command/RemoveNotUniqueHelperTest.java

```diff
@@ -0,0 +1,93 @@
+package cgeo.geocaching.command;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+import junit.framework.TestCase;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import cgeo.geocaching.models.Geocache;
```

Comment:
Please sort the imports as stated in https://github.com/cgeo/cgeo/wiki/Coding-Conventions